oneflow/core/framework/nn_graph.cpp

```diff
@@ -167,13 +175,15 @@ Maybe<void> NNGraph::RegisterVariableOpNamesAndTensors(
     const std::vector<std::string>& variable_op_names,
     const std::vector<std::shared_ptr<one::Tensor>>& variable_tensors) {
   JUST(vm::CurrentRankSync());
-  CHECK_EQ_OR_RETURN(variable_op_names.size(), variable_tensors.size());
+  CHECK_EQ_OR_RETURN(variable_op_names.size(), variable_tensors.size())
+      << "Number of variable names and tensors mismatch";
   for (int32_t i = 0; i < variable_op_names.size(); ++i) {
     const std::shared_ptr<one::Tensor>& var = variable_tensors.at(i);
-    CHECK_OR_RETURN(var->is_eager());
+    CHECK_OR_RETURN(var->is_eager()) << "Tensor variable should be eager";
```

Comment:
```suggestion
    CHECK_OR_RETURN(var->is_eager()) << "Tensor variable should be eager";
```

这里的报错是不是可以多一些限定，也就是“这种XX情况下，Tensor Variable 应该是 Eager 的”

因为我感觉现在的这种说法，会让人误解，以为所有的 variable 必须是 eager

Reply:
已增加限定